notfound-org.php

```diff
@@ -3,8 +3,8 @@
 Plugin Name: NotFound.org 404 Page
 Plugin URI: https://github.com/katzgrau/wp-notfound-org
 Description: A Wordpress plugin that turns your 404 "not found" error into a page devoted to helping find missing children.
-Version: 1.1.0
-Author: Kenny Katzgrau
+Version: 1.2.0
+Author: Kenny Katzgrau & Arno Van Biesen
```

Comment:
Maybe add to a README under contributors section?

Reply:
Updated the readme

Reply:
Hi @ArnoVanBiesen - I think we might want to revert this one since you're now in the contributors section. Is there a reason you want to add that?

Reply:
New commit has been made